src/EcommerceStatsBundle/Bridge/Chameleon/Migration/Script/update-1616421500.inc.php

```diff
@@ -0,0 +1,18 @@
+<h1>Build #1616421500</h1>
+<h2>Date: 2020-09-25</h2>
+<div class="changelog">
+    - ref #636: Add routing definitions
+</div>
+<?php
+$data = TCMSLogChange::createMigrationQueryData('pkg_cms_routing', 'de')
```

Comment:
This now works (in the backend)?

Reply:
The only thing that doesn't work in the backend is generation of the routes added through `pkg_cms_routing` through `UrlGenerator`, which is why the module generates it manually with the help of `UrlUtil` now. The actual routing does work.